src/earthdataVarinfo/handler.py

```diff
@@ -0,0 +1,70 @@
+import os
+import traceback
+import json
+import shutil
+from tempfile import mkdtemp
+
+from varinfo import VarInfoFromNetCDF4
+from varinfo.cmr_search import get_granules, get_granule_link, download_granule
+from varinfo.umm_var import export_all_umm_var_to_json, get_all_umm_var
+
+
+def main(event, context):
+  # Setup CMR search url from cmrRootUrl
+  cmr_url = os.environ['cmrRootUrl'] + '/search/'
+
+  temp_dir = None
```

Comment:
Nit: This line is unnecessary.

Reply:
Needed in earlier version but no longer needed with Context Manager switch.